src/Sylius/Behat/Page/Admin/ShippingMethod/CreatePage.php

```diff
@@ -13,179 +13,8 @@
 
 namespace Sylius\Behat\Page\Admin\ShippingMethod;
 
-use Behat\Mink\Element\NodeElement;
-use Behat\Mink\Exception\ElementNotFoundException;
-use Sylius\Behat\Behaviour\SpecifiesItsField;
 use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
-use Sylius\Behat\Service\DriverHelper;
-use Sylius\Behat\Service\TabsHelper;
-use Sylius\Component\Core\Formatter\StringInflector;
-use Webmozart\Assert\Assert;
 
 class CreatePage extends BaseCreatePage implements CreatePageInterface
```

Comment:
Both `CreatePage` and `UpdatePage` for shipping method could be probably removed as they are empty now